docs/index.html

```diff
@@ -154,6 +154,7 @@ <h6>Paste that at a Terminal prompt.</h6>
                                 <p>For detailed installation instruction go to: </br><a href="https://trinity-client.readthedocs.io/en/latest/guides/quickstart.html#installation">https://trinity-client.readthedocs.io/en/latest/guides/quickstart.html#installation</a></p>
                                 <p>Quick Start: <a href="https://trinity-client.readthedocs.io/en/latest/guides/quickstart.html">https://trinity-client.readthedocs.io/en/latest/guides/quickstart.html</a></p>
                                 <p>Github: <a href="https://github.com/ethereum/trinity">https://github.com/ethereum/trinity</a></p>
+                                <p>Serenity (Eth 2.0) Beacon Chain implementation: <a href="https://github.com/ethereum/trinity/blob/master/eth2/README.md">https://github.com/ethereum/trinity/blob/master/eth2/README.md</a></p>
```

Comment:
Github supports relative links.  This link can be can be changed to `[Eth2.0 Beacon Chain Implementation](./eth2/README.md)` which github should correctly render (you can test by browsing the README in your banch to see if the link is correcT).